Free-Code-Camp-Front-End-Development-Certificate.md

```diff
@@ -3,8 +3,10 @@
 ![An image of a sample Front End Development Certificate](http://i.imgur.com/UrU2ki8.png)
 
 1. Complete all of our current Basic Algorithms
-1. Complete all of our current Basic Front End Projects
-1. Complete all of our current Intermediate Algorithms
-1. Complete all of our current Intermediate Front End Projects
+2. Complete all of our current Basic Front End Projects
+3. Complete all of our current Intermediate Algorithms
+4. Complete all of our current Intermediate Front End Projects
+5. Complete all of our current Advanced Algorithms
+6. Complete all of our current Advanced Front End Projects
 
-Once you've completed all of the above, you can complete our "Claim your Front End Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.
\ No newline at end of file
+Once you've completed all of the above, you can complete our "Claim your Front End Development Certificate" challenge, which will require you to sign our Academic Honesty Pledge. Then we will instantly issue your verified certificate, which will become accessible from your portfolio page.

```

Comment:
1. Change `Certificate` > `Certification` and elsewhere.
2. Add the link to the challenge to the text `Claim your Front End Development Certificate`